spring-cloud-stream-binders/spring-cloud-stream-binder-rabbit/src/main/java/org/springframework/cloud/stream/binder/rabbit/RabbitMessageChannelBinder.java

```diff
@@ -491,13 +491,7 @@ public void declareQueue(String beanName, Queue queue) {
 			}
 		}
 		catch (UncategorizedAmqpException e) {
-			if (e.getCause() instanceof NullPointerException) {
```

Comment:
I saw that the issue https://jira.spring.io/browse/AMQP-565 is fixed in 1.5.4 and `SCSt-Rabbit-Binder` uses Spring AMQP 1.5.4.